test/e2e/docs/overview.md

```diff
@@ -29,13 +29,18 @@ To accelerate development by being a force for continuous improvement, and help
 
 ## What is tested?
```

Comment:
Do we want to relabel this section "Test Groups" to make it more accurate to the content?

Reply:
See below - I've simplified this section to a broad overview, and moved the nitty gritty about the groups into Running Tests in CI with 66fb178ec3954b31f54b1479641a438685875618.